src/ducks/components/SmallAppItem.jsx

```diff
@@ -1,20 +1,23 @@
 import React from 'react'
 
-const SmallAppItem = ({ slug, developer, editor, icon, name, version, onClick }) => (
-  <div className='sto-small-app-item' onClick={onClick} >
-    <img src={icon} alt={`${slug}-icon`} width='64' height='64' className='sto-small-app-item-icon' />
-    <div className='sto-small-app-item-desc'>
-      <h4 className='sto-small-app-item-title'>
-        {editor && `${editor} `}{name}
-      </h4>
-      <p className='sto-small-app-item-detail'>
-        {developer.name === 'Cozy' ? 'Cozy Cloud Inc.' : developer.name}
-      </p>
-      <p className='sto-small-app-item-detail'>
-        {version && `Version ${version.match(/^(.*)-.*$/)[1]}`}
-      </p>
+const SmallAppItem = ({ slug, developer, editor, icon, name, version, onClick }) => {
+  const appVersion = version && (version.match(/^(.*)-.*$/) ? version.match(/^(.*)-.*$/)[1] : version)
+  return (
+    <div className='sto-small-app-item' onClick={onClick} >
+      <img src={icon} alt={`${slug}-icon`} width='64' height='64' className='sto-small-app-item-icon' />
+      <div className='sto-small-app-item-desc'>
+        <h4 className='sto-small-app-item-title'>
+          {editor && `${editor} `}{name}
+        </h4>
+        <p className='sto-small-app-item-detail'>
+          {developer.name === 'Cozy' ? 'Cozy Cloud Inc.' : developer.name}
```

Comment:
I think we should never hardcode UI text in our code. We should always use translations instead.

Reply:
I agree, but it's an exception I think, we will always display it like this, won't we?

Reply:
By the way why do we have to do this ? IMHO this is an information that should be in the manifest whatever happen, don't you think ? Even if we need to add a `displayName` property to `developer`.

Reply:
I agree, I hope that will be a temporary code here. But by waiting to be in our manifests as developer name, it does the job :/